### ISSUE TITLE
Wizard

### ISSUE THREAD

**Issue by Marpingen** [QGEP/QGEP#238]
Nach der Installation des Plugins QGEP kann ich nicht auf die Bottoms wie z.B SQL, Wizard klicken. Diese bleiben immer grau hinterlegt.

**Comment by m-kuhn** [comment]
Wurde die [Demo Datenbank installiert und das Demo Projekt geöffnet](http://qgep.github.io/docs/de/installation-guide/workstation.html#install-the-demo-project)?

**Comment by Marpingen** [comment]
Ja wurde installiert und geöffnet

**Comment by Marpingen** [comment]
![pic](https://cloud.githubusercontent.com/assets/23099048/19770162/8ccbec8c-9c5e-11e6-8a2e-5fe68e46455b.png)

**Comment by sjib** [comment]
Versuchen Sie QGIS auf English laufen zu lassen statt auf Deutsch

**Comment by Marpingen** [comment]
Funktioniert. Aber ohne einen Informatiker hätte ich es nicht geschafft. Viele kleine Probleme. Aber nach einer Woche Installation geschafft. Klasse Arbeit auch von euch!!! Danke

**Comment by m-kuhn** [comment]
Danke für die Rückmeldung. Wir wären sehr dankbar über Infos, die den Installationsprozess vereinfachen könnten (Bzgl. Dokumentation oder auch sonst).

**Comment by Marpingen** [comment]
Also ich sage mal, es musste sehr viel ausprobiert werden. Es gab aber hauptsächlich bei der Installation des Plugins probleme. Wir haben eine kleines Bat-Programm geschrieben, welche das Plugin letztendlich funktiontüchtig gemacht hat. Ein Fazit ist auf jeden Fall das es z.B. mit der QGIS Version 2.18 nicht einwandfrei funktioniert hat. Erst bei Umstellung auf die 2.14.8 Version hat es geklappt. 

Aber ich merke gerade das einige Sachen noch nicht funtionieren. Derzeit muss alles händisch eingetragen werden. Wir suchen nun nach Möglichkeiten Informationen möglichst automatisiert einzugeben. Auch einzelne Kanalschächte die wir vorher schon eingeladen haben, können nicht weiter bearbeitet werden. Also wir haben noch ein bisschen Arbeit vor uns. 

Was hat es eigentlich mit der Datei/Anleitung GEPView-Howto 1.2 Chapter 4.1 auf sich? Wo finde ich diese?

Werde über meinen Fortschritt jedoch berichten... Vorallem

**Comment by sjib** [comment]
Was hat es eigentlich mit der Datei/Anleitung GEPView-Howto 1.2 Chapter 4.1 auf sich?
Das ist die alte Dokumentation des Vorläuferprojektes GEPView, von dem wir viele Arbeitsschritte übernommen haben. Die Verweise sind nur darum drin, um allenfalls die Doku mit Informationen von dort zu ergänzen, falls dies noch Sinn macht. Werden wir mit der Zeit aufräumen, darum einfach drin lassen.

**Comment by sjib** [comment]
Ja, QGEP ist so vorbereitet, dass man alles auch umstellen kann bezüglich Sprache - ist noch in der Pipeline, solange der [Projektgenerator] (https://github.com/QGEP/QGEP/issues/156) noch nicht fertig ist.

**Comment by Marpingen** [comment]
Hallo melde mich wieder zu Wort. 
Uns ist aufgefalles das die Datenbank verschiedene ID (identifier) benutzt bzw. bezieht. Jedoch werden aus dem Quelltext heraus die Bezüge nicht  richtig dargestellt. Mit einem kleine Hilfsquelltext konnten wir dieses Problem (kurzzeitig) umgehen.

Meine Frage ist jedoch eine andere. Bezgl. der Tiefe vom Kanalschacht. Ich sehe zwar, dass die Kanalschachttiefe berechnet wird ,aber jedoch in der Ausgabe (Formular) nicht sichtbar ist. Sprich, die Differenz zwischen Sohlhöhe und Deckelhöhe. Ist Ihnen da etwas bekannt?

**Comment by sjib** [comment]
Nein, haben wir bisher nicht auf dem Formular - da dort nicht Differenzen angezeigt sondern nur die Attribute die man eingibt angezeigt werden.

**Comment by Marpingen** [comment]
Entschuldigen Sie.  Bei manhole (Kanalschacht) kann man bei Tiefe (depth) nur integer Werte eingeben. Ist dies bewusst so gewählt?

**Comment by Marpingen** [comment]
![tiefe 1 int](https://cloud.githubusercontent.com/assets/23099048/20528133/4a534c98-b0cb-11e6-900f-54481b776993.PNG)
![tiefe 2 int](https://cloud.githubusercontent.com/assets/23099048/20528132/4a50f8c6-b0cb-11e6-83a4-1ce1271d1325.PNG)

**Comment by m-kuhn** [comment]
Die Schachttiefe wird in [mm] eingegeben (siehe auch Kommentar auf Screenshot 1 aus PgAdmin)

**Comment by Marpingen** [comment]
Sorry mein Fehler :+1:

**Comment by urskaufmann** [comment]
Ich habe das mit Stefan Burckhardt im Oktober diskutiert. 
In der Praxis wird die Tiefe in m angegeben (als Differenz der Koten Deckel - Kote Sohle). Und nicht in mm wie die übrigen Schachtmasse (Dim1, Dim2). 
Die Tiefe ist nicht ein Attribut im Datenmodell.
Die Quintessenz war dann:
"Gut, dann kann man die Einheit im QGEP Code ändern und ich muss nichts an der VSA-DSS ändern.
Gruss Stefan"
Diese Änderung im QGEP Code hat bisher offenbar noch niemand gemacht. Wäre richtig, dass zu ändern (Matthias)

**Comment by Marpingen** [comment]
![tiefe 2 int](https://cloud.githubusercontent.com/assets/23099048/20601987/ade04932-b25b-11e6-95a0-a99f52de1bda.PNG)

**Comment by Marpingen** [comment]
Hallo, melde mich wieder zu Wort.

Sagt dieser Fehler jemanden was.

Aufrund des mehrfachen Wortaustauschs würde ich mich gerne präsentieren: Ich schreibe derzeit meine Master-Arbeit und versuche derzeit ein Kanalkataster für eine Gemeinde zu erstellen. Je nachdem wie das hier läuft werden sich wahrscheinlich noch mehr Gemeinden einbinden wollen.
Ihr macht einen Klasse Job!!!

**Comment by m-kuhn** [comment]
Dieser Fehler kommt - wenn ich mich richtig erinnere - wenn eine falsche Kombination von Datenbank und Projekt verwendet wird, am Besten nochmal mit dem aktuellsten Schema versuchen.

Danke für die Vorstellung. Ich würde mich auch über eine kurze Mail freuen, damit wir auch einen direkten Draht haben.

**Comment by Marpingen** [comment]
Gerne werde ich dies tun.
Habe über die Datenbank bisher die einzelnen Punkte (Schächte) mit Informationen gefüllt. Bin nun dabei die Haltungen (leider manuell) zu digitalisieren.

Ich denke das es nur mit großer Anstrenung möglich ist die Haltungen automatisch einzubinden. Allein das Problem der Fließrichtung macht es schon schwierig nehme ich an. 

Eine Sache wundert mich jedoch. Ich habe im od. wastewater_structure ja die Deckel und Sohlhöhe (From-level-  To_level) angegeben. Die Sohlhöhe wird jedoch beim Digitalisieren der Haltungen nicht übernommen und muss nochmal manuell eingetippt werden übernommen. Ist die bewusst getrennt worden?

Das selbe Problem leider auch mit der hydraulichen und hierarchischen Funktion.Diese habe ich den Schächten auch zugeordnet und muss diese nun auch manuell eingeben. Vielleicht können Sie weiter helfen.

**Comment by m-kuhn** [comment]
>  Die Sohlhöhe wird jedoch beim Digitalisieren der Haltungen nicht übernommen und muss nochmal manuell eingetippt werden übernommen. Ist die bewusst getrennt worden?

Solche "kleine Hilfen" könnte man wohl noch viele machen. Am besten hier einen neuen Issue eröffnen, damit wir das individuell weiterverfolgen können (am liebsten auf Englisch, damit auch das welsche Publikum mitdiskutieren kann).

> Das selbe Problem leider auch mit der hydraulichen und hierarchischen Funktion.Diese habe ich den Schächten auch zugeordnet und muss diese nun auch manuell eingeben. Vielleicht können Sie weiter helfen.

Über die Eingabemasken oder direkt in der Datenbank? Wenn ich nicht falsch liege, sollte diese für Schächte nicht eingegeben werden, sondern automatisch aus den angeschlossenen Kanälen gesetzt werden, ist aber für die Darstellung wichtig...

**Comment by Marpingen** [comment]
Vielen Dank. Ok. Ich weiss bescheid.

**Comment by Marpingen** [comment]
Ich hätte bitt noch eine kurze Frage. Was kann ich mit diesem Button tun. Was bewirkt es und inwiefern ist es hilfreich? (connenct wastewater network element)

![vernetzen](https://cloud.githubusercontent.com/assets/23099048/20801463/5119e864-b7e9-11e6-8e75-eb759707c72a.PNG)

**Comment by m-kuhn** [comment]
das ist ansatzweise [hier dokumentiert](http://qgep.github.io/docs/de/user-guide/editing/connect_wastewater_network_elements.html) verbesserungsvorschläge sind sehr erwünscht :)

**Comment by Marpingen** [comment]
Das hatt ich schon gesehen. Wollte nur wissen was das bewirken soll :). Also Abwassertechnisch.

**Comment by m-kuhn** [comment]
Dabei geht es um die topologische Verknüpfung zwischen Elementen.

Die Verknüpfung zwischen Haltungen und Abwasserknoten (sind beides Abwassernetzelemente), resp. auch das Anknüpfen von Einzugsgebieten an diese Elemente findet auf der logischen Ebene über Fremdschlüssel (fk_wastewater_networkelement) statt (und nicht über die geometrische Lage der Elemente).

Dieses Tool kann benutzt werden, um diese Fremdschlüssel korrekt zu setzen.
Beispielsweise, falls bei der Netzverfolgung irgendwo fehlende Verbindungen festgestellt wurden.

Wie gesagt, [Verbesserungsvorschläge erwünscht](
https://github.com/QGEP/docs/edit/master/en/user-guide/editing/connect_wastewater_network_elements.rst) ;)

**Comment by Marpingen** [comment]
![netzverfolgung](https://cloud.githubusercontent.com/assets/23099048/21258060/98e9bdac-c37b-11e6-97d2-e6fb06c7a87f.PNG)


Ich habe gehofft, dass das Symbol mein Problem lösen könnte, aber vergebens. Problem schildere ich anhand des Bildes. Wenn ich z.B. den Schacht "1" direkt mit Schacht "2" Verbinde und als Zwischenkoordinate den Punkt "b" (Stutze) mit einbinde und anschliessend den Schacht "3" mit der Stutze "b" verbiinde habe ich folgendes Problem. Wenn ich nun bei Punkt 2 die Netzvervolgung abwärts suchend einschalte, zeigt er mir einen geleben Strang an welcher bis zur eins führt und lässt aber den Verlauf von 3 zu b aus.  Sobald ich einen Punkt praktisch nur als zwischenpunkt einbringe, wird dieser trotz Verbindung zu einem anderen Punkt nicht beachtet. Wenn ich nun die Netzverfolgung bei Punkt 3 anklicke wird es mir nur bis zum Punkt b angezeigt. Gibt es da eventuell eine Lösung?

LG

**Comment by m-kuhn** [comment]
Hängt evtl zusammen mit https://github.com/QGEP/QGEP/issues/248
